platforms/hyperledger-indy/configuration/roles/setup/identity/tasks/nested_main.yaml

```diff
@@ -0,0 +1,119 @@
+- name: Select Admin Identity for Organisation {{ component_name }}
+  shell: |
+  
+    selectedAdmin=""
+    first_global_trustee=""
+    admin_Org=""
+    global_Org=""
+    admin_url=""
+    global_url=""
+    {% if network['organizations'] is defined %}
+      {% for organization in network['organizations'] %}
+        first_trustee_in_org=""
```

Comment:
rename the variable to e.g. first_admin_in_org as you use it for trustees + stewards